src/panels/lovelace/components/hui-card-options.ts

```diff
@@ -76,7 +76,7 @@ export class HuiCardOptions extends LitElement {
                   "ui.panel.lovelace.editor.edit_card.move"
                 )}</paper-item
               >
-              <paper-item @click="${this._deleteCard}">
+              <paper-item class="delete-item" @click="${this._deleteCard}">
```

Comment:
You will have to resolve the build error by replacing `class="delete-item"` with `·class="delete-item"`. Thanks :)

Reply:
> You will have to resolve the build error by replacing `class="delete-item"` with `·class="delete-item"`. Thanks :)

Fixed! Thanks for the feedback

Reply:
Great! No problem :)